lua/treesitter-context.lua

```diff
@@ -165,7 +167,6 @@ function M.enable()
 
   if config.multiwindow then
     table.insert(update_events, 'WinResized')
-    table.insert(update_events, 'WinLeave')
```

Comment:
I can't recall why I used this event for multi-window support. It appears to be unnecessary because it triggers for the window being left, and nvim_get_current_win() returns the window that the user is exiting.